flang/lib/Lower/OpenMP.cpp

```diff
@@ -533,8 +533,7 @@ static void genOMP(Fortran::lower::AbstractConverter &converter,
 
   fir::FirOpBuilder &firOpBuilder = converter.getFirOpBuilder();
   mlir::Location currentLocation = converter.getCurrentLocation();
-  SmallVector<Value, 4> lowerBound, upperBound, step, privateClauseOperands,
-      firstPrivateClauseOperands, lastPrivateClauseOperands, linearVars,
+  SmallVector<Value, 4> lowerBound, upperBound, step, linearVars,
```

Comment:
```suggestion
  llvm::SmallVector<Value, 4> lowerBound, upperBound, step, linearVars,
```

Reply:
nit: In lowering, the namespace is specified to be consistent and avoid class name conflicts and/or errors.